dockerHashing/src/main/scala/cromwell/docker/registryv2/DockerRegistryV2Abstract.scala

```diff
@@ -121,8 +127,12 @@ abstract class DockerRegistryV2Abstract(override val config: DockerRegistryConfi
     * @return docker info response
     */
   protected def getDockerResponse(token: Option[String], dockerInfoContext: DockerInfoContext)(implicit client: Client[IO]): IO[DockerInfoSuccessResponse] = {
-    val request = manifestRequest(token, dockerInfoContext.dockerImageID)
-    executeRequest(request, handleManifestResponse(dockerInfoContext, token))
+    val requestV2 = manifestRequest(token, dockerInfoContext.dockerImageID, AcceptDockerManifestV2Header)
+    def requestV1 = manifestRequest(token, dockerInfoContext.dockerImageID, AcceptOCIIndexV1Header)
```

Comment:
Is there a reason we `val` one and `def` the other?

Reply:
So I made the second a def to essentially only build the second manifest request if it gets called. This may be unnecessary, but I was trying to avoid the case where we always construct both requests but only use the first.

Reply:
I think `lazy val` is more common for situations like this.